src/Uno.UWP/ApplicationModel/Background/BackgroundTaskDeferral.Android.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Windows.ApplicationModel.Background
+{
+	public partial class BackgroundTaskDeferral
+	{
+		/// <summary>
+		/// On Android, BackgroundTaskDeferral has no meaning (background tasks will not be cancelled by OS, unless it takes really much time)
+		/// </summary>
+		public void Complete()
+		{
+			// on Android, there is nothing to do
+		}

```

Comment:
It's not the only platform where it has no meaning.

Reply:
Maybe. But I have no iOS device, no macOS device... I have (besides UWP) only Android device (now in servicing, as accumulator is dead). And while porting my UWP apps to Android, I'm forced to use

#if` NETFX_CORE
BackgroundTaskDeferral.Complete();
#endif

So this small addition makes porting of apps easier for me.